streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/StickyTaskAssignorTest.java

```diff
@@ -151,6 +151,40 @@ public void shouldAssignBasedOnCapacity() {
         assertThat(clients.get(p2).activeTasks().size(), equalTo(2));
     }
 
+    @Test
+    public void shouldAssignTasksEvenlyWithUnequalTopicGroupSizes() {
+        final TaskId task10 = new TaskId(1, 0);
```

Comment:
Why recreating these objects? They are existed above: https://github.com/apache/kafka/pull/5390/files#diff-3cf77a4a8be4dc65221a87377c76ad33R52

Reply:
Just to clarify, you do not need to create 12 tasks to reproduce the issue, just task10 and task00..05 should be sufficient..

Reply:
My preference is to have a test that replicates the original conditions, but I'll update it.